packages/openapi3/test/circular-references.test.ts

```diff
@@ -1,53 +1,56 @@
 import { deepStrictEqual } from "assert";
 import { describe, it } from "vitest";
-import { oapiForModel } from "./test-host.js";
+import { OpenAPISpecHelpers } from "./test-host.js";
 
-describe("openapi3: circular reference", () => {
-  it("can reference itself via a property", async () => {
-    const res = await oapiForModel(
-      "Pet",
-      `
+describe.each(Object.values(OpenAPISpecHelpers))(
```

Comment:
wondering if we either make an helper for this construct or try to shorten things so the describe can remain on one line and this doesn't need extra indent for all files